app/src/main/java/org/oppia/app/settings/profile/ProfileListFragment.kt

```diff
@@ -0,0 +1,34 @@
+package org.oppia.app.settings.profile
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.app.fragment.InjectableFragment
+import javax.inject.Inject
+
+/** Activity that allows users to select a profile to edit from settings. */
```

Comment:
This kdoc is incorrect because you are talking about the next screens. Instead comment should explain about current screen only.
`/** Activity to display all profiles to admin. */`

Reply:
Changed to `/** Fragment to display all profiles to admin. */`  
Also changed the kdoc in `ProfileListActivity` to `/** Activity to display all profiles to admin. */`